docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -20,12 +20,35 @@ For this purpose, *Fast DDS Statistics module* exposes an extended DDS |Statisti
 Enable statistics DataWriters
```

Comment:
I suggest reordering the information. This section should include the information about the new method `enable_statistics_datawriter_with_profile`. But there is another section (`Automatically enabling statistics DataWriters`) where the builtin statistics datawriters are dealt with. Instead of repeating here the information already explained there, it makes sense to extend also that specific section.